src/harness/rwcRunner.ts

```diff
@@ -43,6 +44,10 @@ module RWC {
                 baselineOpts = undefined;
                 baseName = undefined;
                 currentDirectory = undefined;
+                // useDefaultLibFile is a flag specified in the json object to indicate whether to use built/local/lib.d.ts
+                // or to use lib.d.ts inside the json object. If the flag is true or undefine, use the build/local/lib.d.ts
+                // otherwise use the lib.d.ts from the json object.

```

Comment:
undefine -> undefined

Also, that is very weird; we're sure that it's *`true`* or `undefined`? Not `false` or `undefined`?